ci/checks/headers_test.sh

```diff
@@ -10,7 +10,7 @@ DIRNAMES="cudf cudf_test"
 
 # existence tests for lib${LIBNAME}
 for DIRNAME in ${DIRNAMES[@]}; do
-    HEADERS=`cd cpp && find include/${DIRNAME}/ -type f \( -iname "*.h" -o  -iname "*.hpp" \) -printf "        - test -f \\\$PREFIX/%p\n" | sort`
+    HEADERS=`cd cpp && find include/${DIRNAME} -type f \( -iname "*.h" -o  -iname "*.hpp" \) -print | sed 's|^|        - test -f $PREFIX/|' | sort`
```

Comment:
Should the $PREFIX still be escaped? Unless I'm misunderstanding, the old code would output `\$PREFIX` whereas the new code just prints `$PREFIX`. The meta.yaml files don't contain the backslash though so I suspect that I just don't know the escaping syntaxes here all too well.

Reply:
single quotes as I use here inhibit shell expansion, which is what is wanted, AFAICT.

Reply:
Ah I didn't see that you had changed the type of quotes. That makes sense then.